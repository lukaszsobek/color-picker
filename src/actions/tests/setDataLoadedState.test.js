import { setDataLoadedState } from "../";

describe("Set loading state of data",() => {

    it("is `loading` by default",() => {
        expect(setDataLoadedState()).toMatchSnapshot();
    });

    it("is `blah` when set to blah",() => {
        expect(setDataLoadedState("blah")).toMatchSnapshot();
    });    
})