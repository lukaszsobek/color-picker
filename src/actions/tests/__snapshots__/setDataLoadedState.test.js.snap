// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Set loading state of data is \`blah\` when set to blah 1`] = `
Object {
  "loadState": "blah",
  "type": "SET_DATA_LOADED_STATE",
}
`;

exports[`Set loading state of data is \`loading\` by default 1`] = `
Object {
  "loadState": "loading",
  "type": "SET_DATA_LOADED_STATE",
}
`;
